packages/block-editor/src/components/block-edit/with-block-bindings-support.js

```diff
@@ -283,40 +247,13 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 		);
 
 		return (
-			<>
-				<BlockEdit
-					{ ...props }
-					attributes={ { ...props.attributes, ...boundAttributes } }
-					setAttributes={ _setAttributes }
-					context={ { ...context, ...updatedContext } }
-				/>
-			</>
+			<BlockEdit
+				{ ...props }
+				attributes={ { ...props.attributes, ...boundAttributes } }
+				setAttributes={ _setAttributes }
+				context={ { ...context, ...updatedContext } }
```

Comment:
Maybe we can move the logic to amplify the context [here](https://github.com/WordPress/gutenberg/blob/5c4ff54d2c682e0a86810071be8944be8ee2d5b9/packages/block-editor/src/components/block-edit/edit.js#L60-L68) and run `withBlockBindingsSupport` after that one?

Reply:
It's all complicated with React hooks, as you can't run them conditionally. The solution is usually to add some additional components that run the hook, but they are rendered conditionally 🤷🏻‍♂️ 

We can optimize that later in the follow up PR.

Reply:
You're right. Let's see if it makes sense to optimize it in a follow-up PR.